kura/emulator/org.eclipse.kura.emulator.clock/META-INF/MANIFEST.MF

```diff
@@ -14,5 +14,5 @@ Import-Package: org.eclipse.kura;version="[1.0,2.0)",
  org.osgi.framework;version="1.5.0",
  org.osgi.service.component;version="1.2.0",
  org.osgi.service.event;version="1.3.0",
- org.slf4j;version="1.6.4"
```

Comment:
What is the reason for this change? previously this import allows all `org.slf4j` versions from 1.6.4 onwards, this reduces the range from 1.7.36 to 1.8.0 excluded.

Reply:
I restricted the versions to max. 1.8 because we are using `log4j-slf4j-impl`, which only supports slf4j < 1.8 (https://github.com/apache/logging-log4j2/blob/release-2.x/RELEASE-NOTES.md#apache-log4j-2190-release-notes).